B827EBFFFEEBE474.json

```diff
@@ -0,0 +1,18 @@
+{
+    "gateway_conf" : {
+         "gateway_ID" : "B827EBFFFEEBE474",
+         "servers" : [
+              {
+                   "server_address" : "us-west.thethings.network",

```

Comment:
This is not the correct address for the router. Check this list for valid routers: https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses